tooling/nargo_cli/src/cli/prove_cmd.rs

```diff
@@ -138,11 +139,11 @@ pub(crate) fn prove_package(
         Format::Toml,
     )?;
 
-    let proof = backend.prove(&compiled_program.circuit, solved_witness)?;
+    let proof = backend.prove(&compiled_program.program, WitnessStack::from(solved_witness))?;
```

Comment:
@TomAFrench As we weren't testing `nargo` on aztec-packages this didn't come up before, but here I do a conversion into a `WitnessStack` so it matches what the backend expects.